src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/Internal/HostTests.cs

```diff
@@ -1215,6 +1287,8 @@ public void Dispose()
             }
         }
 
+
```

Comment:
nit: extra lines

Reply:
I'll remove in a separate PR (https://github.com/dotnet/runtime/pull/40401). Thanks @Anipik for noticing